{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/datacapturing\/serverconnectionerror\/errordescription-9jvu2"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/DataCapturing\/documentation\/DataCapturing\/ServerConnectionError\/errorDescription-9jvu2","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"The internationalized error description providing further details about a thrown error."}],"kind":"symbol","metadata":{"role":"symbol","title":"errorDescription","roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"symbolKind":"property","externalID":"s:13DataCapturing21ServerConnectionErrorO16errorDescriptionSSSgvp","extendedModule":"DataCapturing","modules":[{"name":"DataCapturing"}]},"hierarchy":{"paths":[["doc:\/\/DataCapturing\/documentation\/DataCapturing","doc:\/\/DataCapturing\/documentation\/DataCapturing\/ServerConnectionError","doc:\/\/DataCapturing\/documentation\/DataCapturing\/ServerConnectionError\/LocalizedError-Implementations"]]},"references":{"doc://DataCapturing/documentation/DataCapturing/ServerConnectionError/LocalizedError-Implementations":{"role":"collectionGroup","title":"LocalizedError Implementations","abstract":[],"identifier":"doc:\/\/DataCapturing\/documentation\/DataCapturing\/ServerConnectionError\/LocalizedError-Implementations","kind":"article","type":"topic","url":"\/documentation\/datacapturing\/serverconnectionerror\/localizederror-implementations"},"doc://DataCapturing/documentation/DataCapturing/ServerConnectionError":{"role":"symbol","title":"ServerConnectionError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ServerConnectionError"}],"abstract":[{"type":"text","text":"A structure encapsulating errors used by server connections."}],"identifier":"doc:\/\/DataCapturing\/documentation\/DataCapturing\/ServerConnectionError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ServerConnectionError"}],"url":"\/documentation\/datacapturing\/serverconnectionerror"},"doc://DataCapturing/documentation/DataCapturing":{"role":"collection","title":"DataCapturing","abstract":[],"identifier":"doc:\/\/DataCapturing\/documentation\/DataCapturing","kind":"symbol","type":"topic","url":"\/documentation\/datacapturing"},"doc://DataCapturing/documentation/DataCapturing/ServerConnectionError/errorDescription-9jvu2":{"role":"symbol","title":"errorDescription","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[{"type":"text","text":"The internationalized error description providing further details about a thrown error."}],"identifier":"doc:\/\/DataCapturing\/documentation\/DataCapturing\/ServerConnectionError\/errorDescription-9jvu2","kind":"symbol","type":"topic","url":"\/documentation\/datacapturing\/serverconnectionerror\/errordescription-9jvu2"}}}